components/caseStudyDetails/CaseStudyDetailHero.tsx

```diff
@@ -25,23 +25,23 @@ const CaseStudyDetailHero = ({ project }: { project: string }) => {
       <div className="flex flex-col items-center gap-6 text-center sm:gap-[3.5rem] 2xl:mx-auto 2xl:max-w-[90rem]">
         <div className="flex flex-col gap-[0.62rem] sm:gap-[1.88rem]">
           <p className="text-sm font-semibold uppercase not-italic leading-[1.1375rem] tracking-[0.2625rem] text-primaryLight dark:text-primaryDark xl:text-[1.25rem]">
-            {projectDetail.type}
+            {projectDetail?.type}
           </p>
           <h1 className="mobileHeading1 dark:text-white900 xl:text-[4rem] xl:leading-[5.2rem] xl:tracking-[-0.04rem]">
             <span className="relative inline-block">
               <span className="absolute top-[1.88rem] h-[1.1875rem] w-full shrink-0 bg-accentSecondary xl:top-[3.3rem] xl:h-[1.757rem]" />
-              <span className="relative">{projectDetail.name}</span>
+              <span className="relative">{projectDetail?.name}</span>
             </span>
             {" - "}
-            {projectDetail.description}
+            {projectDetail?.description}
           </h1>
         </div>
         <Image
           className="sm:h-full sm:max-h-[21.875rem] sm:w-full sm:max-w-[46.375rem]"
           width={345}
           height={163}
-          src={projectDetail.image}
-          alt={`An image of ${projectDetail.name}`}
+          src={projectDetail?.image}
+          alt={`An image of ${projectDetail?.name}`}
```

Comment:
In this component there is too many divs. Can you replace some with semantic html elements?